crawl (1:4.0.0beta24-6.kon.1a) local; urgency=medium

  * Local version!
  * Compile with GCC 3.2, to fix segfault on Alpha.

 -- Kalle Olavi Niemitalo <kon@iki.fi>  Sat,  8 Mar 2003 18:51:09 +0200

diff -ru crawl-4.0.0beta24-6/debian/control crawl-4.0.0beta24-6.kon.1/debian/control
--- crawl-4.0.0beta24-6/debian/control	Sat Mar  8 18:48:28 2003
+++ crawl-4.0.0beta24-6.kon.1/debian/control	Sat Mar  8 18:15:31 2003
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Guus Sliepen <guus@debian.org>
 Standards-Version: 3.5.2
-Build-Depends: debhelper (>= 2.0.40), g++, libncurses5-dev
+Build-Depends: debhelper (>= 2.0.40), g++-3.2, libncurses5-dev
 
 Package: crawl
 Architecture: any
diff -ru crawl-4.0.0beta24-6/debian/rules crawl-4.0.0beta24-6.kon.1/debian/rules
--- crawl-4.0.0beta24-6/debian/rules	Sat Mar  8 18:48:28 2003
+++ crawl-4.0.0beta24-6.kon.1/debian/rules	Sat Mar  8 18:15:24 2003
@@ -6,7 +6,7 @@
 build-stamp:
 	dh_testdir
 
-	cd source; $(MAKE) all
+	cd source; $(MAKE) all CXX=g++-3.2
 
 	touch build-stamp
 
